elastic/kibana
# [Fleet] Support package capabilities filtering

## Summary

Depends on https://github.com/elastic/package-registry/pull/1061

Pass the capabilities configured in `xpack.fleet.internal.capabilities` to the package registry to allow filtering on capabilities.


## Manual test instructions

* Using that custom package registry you can try to add `elastic.capabilities` to a package and `xpack.fleet.internal.capabilities` to you kibana.dev.yml file to see that filitering is correctly done.



## Todo

- [x] manual test instructions
- [x] Configure capabilities for observability and security project

Closes https://github.com/elastic/kibana/issues/161215
